templates/test/ci/prow-topology/base.yaml

```diff
@@ -0,0 +1,258 @@
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: ClusterClass
+metadata:
+  name: ci-default
+spec:
+  controlPlane:
+    ref:
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+      kind: KubeadmControlPlaneTemplate
+      name: ci-default-kubeadm-control-plane
+    machineInfrastructure:
+      ref:
+        kind: AzureMachineTemplate
+        apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+        name: ci-default-control-plane
+  infrastructure:
+    ref:
+      apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+      kind: AzureClusterTemplate
+      name: ci-default-azure-cluster
+  workers:
+    machineDeployments:
+      - class: ci-default-worker
+        template:
+          bootstrap:
+            ref:
+              apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+              kind: KubeadmConfigTemplate
+              name: ci-default-worker
+          infrastructure:
+            ref:
+              apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+              kind: AzureMachineTemplate
+              name: ci-default-worker
+        machineHealthCheck:
```

Comment:
is it possible to have a machine healthcheck for the windows nodes too? I am not super familiar with that it checks but might be helpful?

Reply:
I've added the same mhc that is being used for linux nodes, which was present in existing prow templates. Is that acceptable for a start?

Reply:
Thanks!